netflix/src/components/Counter.jsx

```diff
@@ -1,8 +1,18 @@
-import React from "react";
+import React, { useState } from "react";
 
-const Counter = () => (
-  <div>
-    <h2>Howdy</h2>
-  </div>
-);
+const Counter = () => {
+  const [count, setCount] = useState(0);
+
+  return (
+    <div className="counter">
+      <button type="button" onClick={() => setCount(count - 1)}>
```

Comment:
Not a bug.
Just to be a better developer, you should always consider your app performance.
Please read this React documentation page with some hints on why writing callbacks into your HTML is not the best idea:
https://reactjs.org/docs/handling-events.html

> The problem with this syntax is that a different callback is created each time the LoggingButton renders. In most cases, this is fine. However, if this callback is passed as a prop to lower components, those components might do an extra re-rendering. We generally recommend binding in the constructor or using the class fields syntax, to avoid this sort of performance problem.